### ISSUE TITLE
DrawerToggle is Unlabeled and Does Nothing for a Screen Reader User

### ISSUE THREAD

**Issue by frastlin** [vaadin/vaadin-app-layout#164]
Hello,
The button in the DrawerToggle is not labeled and it doesn't do anything according to a screen reader.
When you arrow over the element, it says "button". This is not useful.
This should either be hidden with aria-hidden="true" or made with the Details element that expands and collapses like a drawer.
Visually, when you click that button, what happens?
Thanks,

**Comment by web-padawan** [comment]
Hi, thanks for the issue. There is `ariaLabel` property which sets the corresponding attribute. Have you tried using it?

**Comment by frastlin** [comment]
Hello, that could label the button, but then you have a button that's just randomly there and does nothing at the top of the page. Aria label doesn't change the fact that the drawer is only visual.
